packages/ui/src/components/Checkbox/checkbox.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { Checkbox as DefaultCheckbox, CheckboxProps as DefaultCheckboxProps } from 'antd'
+
+const CHECKBOX_CLASS_PREFIX = 'condo-checkbox'
+
+export type CheckboxProps = Pick<DefaultCheckboxProps, 'autoFocus' | 'defaultChecked' | 'disabled' | 'onChange'>
```

Comment:
why there are no `checked`, `children`, `onClick`, `indeterminate` properties?

Reply:
children added

we do not use checked and onClick because onChange and defaultValue exist (also, checked or onClick have not been used in any of the checkboxes in the current version of the project)
indeterminate is also not used neither in the layout nor in the current state of the project

Reply:
Yes, `onClick` is really not used, you are right.

But `checked` is used. (`defaultValue` is only the initial state, as far as I know). 
`intermediate` is used in the new feature (https://github.com/open-condo-software/condo/pull/2672). In the layouts a little later this will be added, I already told the designers about it.

Reply:
In the same feature (https://github.com/open-condo-software/condo/pull/2672), the checkbox label can be bold. Will ui kit make it possible?

Reply:
Check ui kit in figma. Added intermediate states for checkboxes